decidim-core/app/views/layouts/decidim/newsletter_base.html.erb

```diff
@@ -6,7 +6,7 @@
     <%= stylesheet_pack_tag "decidim_email" %>
   </head>
 
-  <%= content_tag :body, class: @preview ? "preview" : nil do %>
+  <body <%= @preview ? 'class="preview"' : nil %>>
```

Comment:
Not sure why, but this is generated like this:

![screenshot of the HTML](https://github.com/decidim/decidim/assets/717367/80e27d40-2b25-4cf8-9319-2e844de421de)

Reply:
I am not very sure, why this is happening. I suspect that we have premailer or some other tool that adds an extra class 
I changed it to : 

```
  <body <%= @preview ? 'class=preview' : nil %>>
```
and everything seems to be ok now .

Reply:
> I suspect that we have premailer or some other tool that adds an extra class 

Oh that makes sense!